docs/How-to-Create-and-Maintain-a-Tap.md

```diff
@@ -16,9 +16,10 @@ See the [manpage](Manpage.md) for more information on repository naming.
 The `brew tap-new` command can be used to create a new tap along with some
 template files.
 
-Tap formulae follow the same format as the core’s ones, and can be added at the
-repository’s root, or under `Formula` or `HomebrewFormula` subdirectories. We
-recommend the latter options because it makes the repository organisation
+Tap formulae follow the same format as the core’s ones, and can be added under
+either the `Formula` or `HomebrewFormula` subdirectories, or at the repository’s
```

Comment:
```suggestion
either the `Formula` subdirectory, `HomebrewFormula` subdirectory or the repository’s
```

Reply:
The extra `the` is fine with me. We avoid the Oxford comma: https://docs.brew.sh/Prose-Style-Guidelines#typographical-conventions